tests/setup/odh-core.yaml

```diff
@@ -91,8 +84,8 @@ spec:
   - kustomizeConfig:
       repoRef:
         name: manifests
-        path: data-science-pipelines-operator/
-    name: data-science-pipelines-operator
```

Comment:
@RobGeada You'll need to rebase and confirm that the kustomizeConfig for `data-science-pipelines-operator` is still in the test kfdef